internal/nearcache/record.go

```diff
@@ -74,27 +73,50 @@ func (r *Record) SetReservationID(rid int64) {
 	atomic.StoreInt64(&r.reservationID, rid)
 }
 
-func (r *Record) LastAccessTimeMS() int64 {
-	return atomic.LoadInt64(&r.lastAccessTime)
+func (r *Record) CreationTime() int64 {
+	t := atomic.LoadInt32(&r.creationTime)
+	return RecomputeWithBaseTime(t)
+}
+
+func (r *Record) SetCreationTime(ms int64) {
```

Comment:
Some fields including creation time are not updated atomically in Java. Why did you update them atomically?

Reply:
Java client uses `volatile` for those, which is not available for Go.